### ISSUE TITLE
Shortcodes

### ISSUE THREAD

**Issue by theoldfellow** [CouchCMS/Documentation#8]
**Page:** `http://docs.couchcms.com/miscellaneous/shortcodes.html`

Google, as they do to protect the guilty, seem to have changed Maps again, so that 'link' button you refer to is no longer there.  I can't find how you can get the required information.

**Comment by cheesypoof** [comment]
Yes @theoldfellow, their layout seems to have changed, so this appears to now be located under `Menu` (top left) > `Share or embed map`. The shortcode should perhaps be updated as well to reflect the new URL structure utilized by Google Maps.

Apologies for the delay...
